ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -254,6 +256,14 @@ interface {{ vlan_interface }}
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if vlan_interfaces[vlan_interface].ipv6_address_virtual is arista.avd.defined %}
```

Comment:
please split these enhancements to `eos_cli_config_gen` out in a separate PR. So we ensure documentation and molecule.